webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Grid,
+  Tab,
+  Tabs,
+  GridItem,
+  Badge,
+  Title,
+  Breadcrumb,
+  BreadcrumbItem,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import Skeleton from 'react-loading-skeleton';
+import TimeAgo from 'react-timeago';
+import StatusAlert from './Status';
+
+import { foremanUrl } from '../../../foreman_tools';
+import { get } from '../../redux/API';
+import { selectAPIResponse } from '../../redux/API/APISelectors';
+
+import Properties from './Properties';
+import ParametersCard from './Parameters';
+import InterfacesCard from './Interfaces';
+import AuditCard from './Audits';
+import ActionsBar from './ActionsBar';
+
+const HostDetails = ({ match }) => {
+  const dispatch = useDispatch();
+  const [activeTab, setActiveTab] = useState(0);
+  const response = useSelector(state =>
+    selectAPIResponse(state, 'HOST_DETAILS')
+  );
+  useEffect(() => {
+    dispatch(
+      get({
+        key: 'HOST_DETAILS',
+        url: foremanUrl(`/api/hosts/${match.params.id}`),
+      })
+    );
+  }, [match.params.id, dispatch]);
+
+  const handleTabClick = (event, tabIndex) => {
+    setActiveTab(tabIndex);
+  };
+  return (
+    <>
+      {response.name ? (
+        <Breadcrumb style={{ marginTop: '15px' }}>
+          <BreadcrumbItem to="/hosts">Hosts</BreadcrumbItem>
+          <BreadcrumbItem isActive>{response.name}</BreadcrumbItem>
+        </Breadcrumb>
+      ) : (
+        <Skeleton />
+      )}
+      {/* TODO: Replace all br with css */}
+      <br />
+      <br />
+      <Grid>
+        <GridItem span={2}>
+          <Title headingLevel="h5" size="2xl">
+            {/* TODO: Make a generic Skeleton HOC (withSkeleton) */}
+            {response.name || <Skeleton />}
+          </Title>
+        </GridItem>
+        <GridItem style={{ marginTop: '5px', marginLeft: '10px' }} span={8}>
+          <Badge key={1}>{response.operatingsystem_name}</Badge>{' '}
+          <Badge key={21}>{response.architecture_name}</Badge>
+        </GridItem>
+        <GridItem span={2}>
+          {response.name ? (
+            <ActionsBar hostName={response.name} />
+          ) : (
+            <Skeleton />
+          )}
+        </GridItem>
+      </Grid>
+      {response.name ? (
+        <Text style={{ fontStyle: 'italic' }} component={TextVariants.p}>
+          {/* TODO: extracting text */}
+          created <TimeAgo date={response.created_at} />
```

Comment:
Why not use `FormattedRelative` from react-intl or Foreman's https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/dates/RelativeDateTime.js ?